modules/virt-deleting-templates-from-custom-namespace.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-deleting-templates-from-custom-namespace.adoc
+
+[id="virt-deleting-templates-from-custom-namespace_{context}"]
+= Deleting templates from a custom namespace
+
+To delete Red Hat templates from a custom namespace, remove the `commonTemplateNamespace` attribute and delete the templates from that custom namespace.
+
+.Procedure
+
+. Open the `hco` CR to edit the file.
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
++
+. Remove the `commonTemplateNamespace` attribute and the custom namespace from the spec.
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  commonTemplatesNamespace: customnamespace <1>
+----
+<1> Delete the `commonTemplatesNamespace` attribute and the value.
++
+. Delete templates from the namespace that was removed from the `hco` CR.
++
+[source,terminal]
+----
+$ oc delete templates -n customnamespace --all
+----
+
+.Verification
+
+* Verify that the templates were deleted from the custom namespace by confirming that templates do not exist in that namespace.
+----
+$ oc get templates -n openshift
```

Comment:
it should be `$ oc get templates -n customnamespace`.

Reply:
Updated the content.